SeedVaultSimulator/src/main/AndroidManifest.xml

```diff
@@ -106,53 +106,55 @@
 
         <!-- NOTE: Order set below that of the privileged alias in case an app holds both
              permissions. Apps normally should hold only one of the two permissions, but the
-             Seed Vault should gracefully handle the case where an app is granted both. These
-             intents are always explicit, so no elevated priority is required. -->
+             Seed Vault should gracefully handle the case where an app is granted both. While these
+             intents should always be explicit, apply elevated priority in case they are ever
+             referenced implicitly. -->
         <activity-alias
             android:name=".authActivityGenericAlias"
             android:exported="true"
             android:permission="com.solanamobile.seedvault.ACCESS_SEED_VAULT"
             android:targetActivity=".ui.AuthorizeActivity">
-            <intent-filter android:order="-100">
+            <intent-filter android:priority="100" android:order="0">
```

Comment:
Adding `priority` is a defensive measure. These `<intent-filter>`s shouldn't be used with implicit Intents; the package or component should always be specified. However, if a caller makes a mistake, adding a `priority` ensures that the system canonical Seed Vault implementation will always be selected.